content/applications/hr/attendances/check_in_check_out.rst

```diff
@@ -0,0 +1,89 @@
+field appears above the button, and the total amount of time that has been logged for the day
+appears beneath that field, in an :guilabel:`HH:MM` (hours:minutes) format.
+
+Click the :guilabel:`Check in` :icon:`fa-sign-in` button to check in. The :guilabel:`🔴 (red
+circle)` in the top menu changes to green, and the widget changes appearance as well. The widget
+updates to reflect that the user has checked in, by changing the green :guilabel:`Check in`
+:icon:`fa-sign-in` button to a yellow :guilabel:`Check out` :icon:`fa-sign-out` button.
+
+Click anywhere on the screen to close the attendance widget.
+
+Check out
+=========
+
+If the user is checking out for the first time, :guilabel:`Since HH:MM (AM/PM)` appears at the top
+of the widget, with the time the user checked in populating the time field. Beneath that line,
+:guilabel:`HH:MM` is displayed, indicating the hours and minutes that have elapsed since checking
+in. As time passes, this value is updated to reflect the hours and minutes that have passed since
+the user checked in.
+
+If the user has previously checked in and out, additional fields are presented. A :guilabel:`Before
+HH:MM (AM/PM)` field appears in addition to the :guilabel:`Since HH:MM (AM/PM)` field. The time
+displayed in both of these fields are populated with the most recent check in time, and will match.
+Beneath the :guilabel:`Before HH:MM (AM/PM)` fiel-d, the previously logged hours are displayed, in an
+:guilabel:`HH:MM` (hours:minutes) format.
+
+In addition, beneath both of these fields, a :guilabel:`Total today` field appears. This field is
+the sum of both the :guilabel:`Before HH:MM (AM/PM)` and :guilabel:`Since HH:MM (AM/PM)` fields, and
+is the total time that will be logged for the user, if they were to log out at that moment.
+
+As time passes, both the :guilabel:`Since HH:MM (AM/PM)` and :guilabel:`Total today` fields are
+updated live. To check out, click the yellow :guilabel:`Check out` :icon:`fa-sign-out` button. The
+attendance widget updates again, displaying the :guilabel:`Total today` field with the logged time,
+and the yellow :guilabel:`Check out` :icon:`fa-sign-out` button changes to a green :guilabel:`Check
+in` :icon:`fa-sign-in` button.
+
+When the user checks out from the database, the *Attendances* app logs the location details for the
+user. This information is **only** logged if the user allows their computer to access this
+information.
+
+.. image:: check_in_check_out/check-in-database-message.png
+   :align: center
+   :alt: The pop-up that appears when an employee checks in inside the database.
+
+.. tip::
+   There is no limit to the amount of times a user can check in and check out. Users are able to
+   check in and out without any time elapsing (a value of 00:00). Each time an employee logs in and
+   out, the information is stored and appears on the dashboard, including check ins and check outs
```

Comment:
would you mind adding a bit more specificity to which dashboard? The main Odoo Dashboard? The main Attendances dashboard? Both? A little elaboration could be super helpful here. Thanks! 🙏